backend/app/lifespan.py

```
@@ -0,0 +1,61 @@
+    
+    yield
+    
+    # Shutdown: Cancel worker task gracefully
+    if worker_task:
+        logger.info("Stopping image upload worker...")
+        worker_task.cancel()
+        try:
+            await worker_task
+        except asyncio.CancelledError:
+            logger.info("Image upload worker stopped")
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)